source/codegen/common_helpers.py

```diff
@@ -204,12 +204,14 @@ def get_input_and_output_custom_types(config, functions):
 
 def _get_nested_types(config, type_name):
     custom_types = set()
-    matching_custom_type = [
```

Comment:
Note: I noticed that this worked for the ni-fake updates I made but not for DC Power. The ni-fake updates had nested type where the nested custom_type was in config.py. DC Power uses NIComplexNumber as a nested type which doesn't show up in config.py but rather is in the common/shared spot nidevice.proto.

So here if it can't find the matching type by name in the `config` just return early. Doesn't affect the generation of any other driver so this should be good.